### DISCUSSION TITLE
FR: In the `Guides` section, link to plugin code which uses that concept

### DISCUSSION THREAD

**Discussion by SkepticMystic** [marcusolsson/obsidian-plugin-docs#36] [Ideas]
For example, in the `Custom Views` section, have a heading at the bottom which links to sections in the code of plugins which have "good" custom views

**Comment by marcusolsson** [comment]
I admit this would be really helpful to have, though I'm a little worried about the maintenance burden.

Would you:

- Permalink to the custom view?
- Link to the latest version of the custom view?
- Link to the plugin?

In all of these cases, we'd have to regularly reevaluate whether they are still “good” custom views. And if we introduce an “In the wild” section for one guide, we'll likely see them in other guides as well. I would struggle to keep these resources up-to-date.

I think I'd prefer if we could distill what makes those custom views "good" and then add it to the guide instead. WDYT?

**Comment by SkepticMystic** [comment]
I get you, I get you.
Although it would be helpful to point to good examples, the maintenance would start piling up.
One small change that comes to mind is to simply make a note of when the example was added:
```
## In The Wild

- [Plugin](repo.git) (2021-11-01)
```
This way people reading would know when last someone updated the note of the example.

> Would you:
> - Permalink to the custom view?
> - Link to the latest version of the custom view?
> - Link to the plugin?

Good question! Given that the author might completely remove the custom view, permalinking could just break (having the date last edited would help here, too). Maybe just linking to the plugin and adding a screenshot of the thing it does well.

But if you decide the admin is too much, then I think distilling the "goodness" into your guide is a fine fallback! Sticking to your guidelines, it would still have to be a working example, and then stuff comes in about copying someone else's code, no? I suppose if the plugin has an appropriate license, that's fine.
Let me know what you decide, I am happy to help contribute using either approach

**Comment by marcusolsson** [comment]
Been thinking a little more about this. If we feel a plugin is a good example, I think we should tell the reader _why_ it's good, rather than pointing to an entire repository, or even a whole file. If we're able to do that, we should instead be able to simplify it, so it's easier to understand.

For example, if we find a really good example we can create a smaller version of it where we focus on the thing we want to show, and then create a page this like one:

https://marcus.se.net/obsidian-plugin-docs/examples/insert-link

We would then be able to link to the plugin, but for credit rather than reference, so there's less need to keep it updated.
